python/tests/datasources/test_cawp_women_legislators.py

```diff
@@ -17,6 +18,13 @@
 
 # UNIT TESTS
 
+def test_pct_never_null():
+    assert pct_never_null(1, 2) == 50
+    assert pct_never_null(0, 0) == 0
+    # TODO uncomment next test once util fn is fixed
```

Comment:
@joshzarrabi when you make your changes to the util, can you also rm this comment and uncomment the last test?

Reply:
Sure!